### DISCUSSION TITLE
是否有办法为各项更新时间附加更复杂的规则？

### DISCUSSION THREAD

**Discussion by abbaccadd0** [vernesong/OpenClash#3810] [Q&A]
或者是否可以自定义更新的corntab？
我想要一天中两个固定时间更新规则，但是选项内没有提供
我尝试修改计划任务中的项目，但是openclash下次启动时会覆盖更改
请问有办法做到这件事吗？还是说我需要去提issues来添加类似的功能？

**Comment by refined-fish** [comment]
数据库和白名单是根据页面设置生成的，只能设置一天一次
